doc/quoting-issues-with-powershell.md

```diff
@@ -1,41 +1,5 @@
 # Quoting issues with PowerShell
 
-**These issues have been fixed in Azure CLI 2.40.0 and PowerShell 7.3. Please update Azure CLI and PowerShell to the latest versions to avoid these issues.**
```

Comment:
We have to roll back the change due to the reverting of `ps1` entry script (https://github.com/Azure/azure-cli/pull/24015).

Reply:
Should we tell user `azps` is kept?

Reply:
`azps` is too buggy at the comment. Let's do that in separate PRs.